### PR TITLE
Add support for EL6

### PR DESCRIPTION
**Pull request by gcymbalski**
Since build infrastructure is transitioning to Travis, EL6 base build
boxes are needed. This setup works locally and pulls in the latest EL6
from Centos in the same manner as 7 has worked in the past.

### CODE REVIEW

File: Dockerfile.el6x [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    make DESTDIR=/app/sparse install && \
+    cd /app/sparse/root && tar cvf /app/sparse/install.tar ./ && \
+    cd - && rm -fr sparse-*
+
+
+# Stage 2, only want the sparse binary
+FROM versity/centos:el6x
+
+# PREFIX defaults to $HOME, so DESTDIR/PREFIX...
+COPY --from=sparse_build /app/sparse/install.tar /app/sparse/install.tar
```
Comment (nichenke) [inline Dockerfile.el6x]: We can nuke the entire sparse bit, it isn't needed for VSM1.

File: Dockerfile.el6x [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+# Use tar so we can skip the existing directory symlinks. Breaking those blows up the world.
+RUN tar -pxvf /app/sparse/install.tar --skip-old-files
+
+# run early, so it gets the right GID that mock would like to use
+RUN groupadd -g 135 mockbuild
```
Comment (nichenke) [inline Dockerfile.el6x]: This is my fault -- mock bits left over. We are not installing mock, so no need to add the user.

**Comment by gcymbalski** [comment]
Hrm, is anything using rpmbuild.sh in here? Looks like the Mock transition may make that worth removing for good.

**Comment by gcymbalski** [comment]
Score, this looks like it; getting mhvtl packages back 👍

File: Dockerfile.el6x (hunk 1 above)
Comment (gcymbalski) [reply Dockerfile.el6x]: Done!

File: Dockerfile.el6x (hunk 2 above)
Comment (gcymbalski) [reply Dockerfile.el6x]: Sounds good; removing that and letting yum do its thing worked fine.

**Comment by nichenke** [comment]
The el7 builds for scoutfs use rpmbuild.sh. My goal is that anything new/updated uses rpmbuild.sh --  mhvtl could, removing some of our custom hacks in that repo and making it closer to upstream.

If we end up needing custom local dependencies or otherwise need the fine grained control of mock, that'd be an argument in its favor.

**Pull request merged**